holmes/common/env_vars.py

```
@@ -38,3 +38,7 @@
 STREAM_CHUNKS_PER_PARSE = int(
     os.environ.get("STREAM_CHUNKS_PER_PARSE", 80)
 )  # Empirical value with 6~ parsing calls. Consider using larger value if LLM response is long as to reduce markdown to section calls.
+
+ROBUSTA_AI_MODEL_NAME_BACKOFF = os.environ.get(
+    "ROBUSTA_AI_MODEL_NAME_BACKOFF", "gpt-4o"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)